src/shared/components/footer/style.css

```diff
@@ -86,3 +92,14 @@
 .inputSubmit:hover {
   background-color: green;
 }
+
+@media mediaSmall {
+  .address {
+    margin-bottom: 15px;
+  }
+
+  .links {
+    margin-bottom: 15px;
+    margin-left: -10px;
```

Comment:
It seems the icon font has some natural padding, better way of tackling this than using negative margin?